core/merginapi.cpp

```diff
@@ -1364,6 +1364,12 @@ void MerginApi::checkMerginVersion( QString apiVersion, bool serverSupportsSubsc
     {
       setApiVersionStatus( MerginApiStatus::INCOMPATIBLE );
     }
+
+    // will be dropped support for old servers (mostly CE servers without workspaces)
+    if ( ( MINIMUM_SERVER_VERSION_MAJOR == major && MINIMUM_SERVER_VERSION_MINOR > minor ) || ( MINIMUM_SERVER_VERSION_MAJOR > major ) )
+    {
+      emit migrationRequested();
+    }
```

Comment:
Unfortunately, this piece of code (`checkMerginVersion`) is executed each time the `GET /ping` request finishes. We need to check this only in the `GET /config` as the ping request is **called several times during the app run**. As a result, this dialogue would show up multiple times.

Reply:
solved